### PR TITLE
Updates Algorithm Arguments Optional

### PR DESCRIPTION
**Pull request by Rafase282**
Updates Algorithm Arguments Optional

Refers to #1094

### CODE REVIEW

File: Algorithm-Arguments-Optional.md [hunk 1]
```diff
@@ -11,25 +13,37 @@ It can be quite complicated to understand what needs to be done. There are alway
 - It has to check if it has one or two arguments passed. More are ignored.
 - If it has only one argument then it has to return a function that uses that number and expects another one, to then add it.
 
-## Hint: 1
+### Relevant Links
+
+- [Link Title 1](http://example.com)
+- [Link Title 2](http://example.com)
```
Comment (atjonathan) [inline Algorithm-Arguments-Optional.md]: Reminder to fix these...

File: Algorithm-Arguments-Optional.md [hunk 2]
```diff
@@ -90,7 +104,13 @@ addTogether(2,3);
 - Still inside the big else, we need to check the argument we saved, if it is a number then we return the function expecting a second argument.
 - Now inside the function we are returning, we have to check for non numbers again just as at the beginning using **checkNum** if undefined then return that, otherwise if numbers add them and return the addition.
 
-## Second Solution:
+#### Relevant Links
+
+- [Link Title 1](http://example.com)
+- [Link Title 2](http://example.com)
```
Comment (atjonathan) [inline Algorithm-Arguments-Optional.md]: Reminder to fix these...

File: Algorithm-Arguments-Optional.md [hunk 3]
```diff
@@ -135,7 +155,17 @@ addTogether(2,3);
...
 
-## Third Solution:
+### Code Explanation:
+
+// TO-DO
+
+#### Relevant Links
+
+- [Link Title 1](http://example.com)
+- [Link Title 2](http://example.com)
```
Comment (atjonathan) [inline Algorithm-Arguments-Optional.md]: Reminder to fix these...

File: Algorithm-Arguments-Optional.md [hunk 4]
```diff
@@ -135,7 +155,17 @@ addTogether(2,3);
 
 :rocket: [Run Code](https://repl.it/CLoA/0)
 
-## Third Solution:
+### Code Explanation:
+
+// TO-DO
```
Comment (atjonathan) [inline Algorithm-Arguments-Optional.md]: Reminder to add content to this...

**Comment by Rafase282** [comment]
Yeah I'm not done yet, I have to look for links, check the solutions and their explanations, which is why I said anyone can pitch in to make things move faster.

**Comment by Rafase282** [comment]
Ready for feedback!

**Comment by atjonathan** [comment]
### **LGTM :+1:**

Let's get one more look from @FreeCodeCamp/wiki-mods

**Comment by koustuvsinha** [comment]
# LGTM :+1: 

Lets :package: :shipit:

**Pull request merged**